db/models.py

```diff
@@ -1 +1,47 @@
-from django.db import models
+from django.db import models as m
```

Comment:
This alias does not look very good. I'd suggest to avoid using it unless it's a convention I'm not aware of.